prow/git/v2/adapter.go

```diff
@@ -86,6 +87,19 @@ func (a *repoClientAdapter) Fetch() error {
 	return errors.New("no Fetch implementation exists in the v1 repo client")
 }
 
+func (a *repoClientAdapter) FetchFromRemote(resolver RemoteResolver, branch string) error {
```

Comment:
Where does this method signature come from? Why do we have a full `resolver`? Why is this useful? What is the method on the `git` interface?

Reply:
I might have missed something in the adapter's implementation. Let me check on this.

Reply:
@stevekuznetsov Are you suggesting to remove this completely from the v1 client?